draft-linus-trans-gossip-ct.md

```diff
@@ -282,7 +284,7 @@ with the following content:
 
   - x509_chain: An array of base64-encoded X.509 certificates. The
     first element is the end-entity certificate, the second chains to
-    the first and so on.
+    the first and so on. [clients may not receive such chains - should this be a requirement? e.g. they may get a bag of certs that admits multiple chains, they may receive chains with missing certs, user might've clicked through a warning etc]
```

Comment:
I think that our intention here was to send the chain that the client constructed, not what was sent in the ServerHello. (Although that would be interesting, it's interesting in a different way and doesn't answer the question of "Why did the client trust this?")

If no such chain is made, and the client clicks through a warning, the spec is undefined.  Perhaps we should define it? (Send the ServerHello, and set a flag indicating no trusted chain was formed?)